sbi/utils/metrics.py

```diff
@@ -20,22 +20,57 @@ def c2st(
     scoring: str = "accuracy",
     z_score: bool = True,
     noise_scale: Optional[float] = None,
+    verbosity: int = 0,
+    clf_class=RandomForestClassifier,
```

Comment:
What's the reason behind this change?

Reply:
Because it is slow as a snail and the randomforest gives the same scores. The Lopez-Paz paper says on this very topic in section 3.3:
> Examples of interpretable features include the filters of the first layer of a neural network, **the feature importance of random forests**, the weights of a generalized linear model, and so on.
The sole purpose of this PR is to increase runtime of c2st by use of a random forest. For more details, see [my study](https://github.com/psteinb/c2st/blob/main/rsc/c2st_results.ipynb) that @janfb asked me to do.

Reply:
MLPs only make sense in the very small dataset regime, i.e. if X[0] < 128. On my machine, the MLP takes 3 seconds here and the RF takes 1. second.

Reply:
looking at the detailed comparison in runtime and accuracy between MLP and RF I think it makes sense to think about changing the default to RF. 
However, I think we still want to be able to change it quickly, so I would vote for changing the kwarg to a `c2st_classifier: str=mlp` or `c2st_classifier: str=rf` and then add both to the doctoring for that `arg` and create the instance of the `clf_class` in the method with an if-else on `c2st_classifier`.

Reply:
then we can work on this PR independent of the discussion about which classifier to use as default.

Reply:
Going to resolve this for the sake of the discussion below.

Reply:
Okey! Fine to use rf then!